scaling_performance/managing_hugepages.adoc

```diff
@@ -0,0 +1,87 @@
+[[scaling-performance-managing-huge-pages]]
+= Managing Huge Pages
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Introduction
+
+[IMPORTANT]
+====
+Managing huge pages is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+In {product-title}, applications in a pod can allocate and consume pre-allocated
+huge pages. This topic describes how.
+
+[[huge-pages-prerequisites]]
+== Prerequisites
+
+. Nodes must pre-allocate huge pages in order for the node to report its huge
+ page capacity. A node can only pre-allocate huge pages for a single size.
+
+. `HugePages` must be set to `true` across the system:
++
+----
+--feature-gates=HugePages=true
+----
+
+[[consuming-huge-pages]]
+== Consuming Huge Pages
+
+Huge pages can be consumed via container level resource requirements using the
+resource name `hugepages-<size>`, where size is the most compact binary
+notation using integer values supported on a particular node. For example, if a
+node supports 2048KiB page sizes, it will expose a schedulable resource
+`hugepages-2Mi`. Unlike CPU or memory, huge pages do not support overcommitment.
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: hugepages-volume-
+spec:
+  containers:
+  - image: fedora:latest
+    command:
+    - sleep
```

Comment:
the cpu manager examples use the pause pod.  we should probably unify these examples.  i suppose sleep infinity is the easiest since it is in the rhel7 base image.